<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>500</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>–§–∏–Ω–∞–Ω—Å—ã</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout">
                <item>
                    <widget class="QPushButton" name="btnAddAccount">
                        <property name="text">
                            <string>–î–æ–±–∞–≤–∏—Ç—å —Å—á—ë—Ç</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="btnDelAccount">
                        <property name="text">
                            <string>–£–¥–∞–ª–∏—Ç—å —Å—á—ë—Ç</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="btnAddTx">
                        <property name="text">
                            <string>–î–æ–±–∞–≤–∏—Ç—å —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏—é</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="btnDelTx">
                        <property name="text">
                            <string>–£–¥–∞–ª–∏—Ç—å —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏—é</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="btnShowImage">
                        <property name="text">
                            <string>–ü–æ–∫–∞–∑–∞—Ç—å –∏–∑–æ–±—Ä–∞–∂–µ–Ω–∏–µ</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="btnShowChart">
                        <property name="text">
                            <string>–ü–æ–∫–∞–∑–∞—Ç—å –≥—Ä–∞—Ñ–∏–∫</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QLabel" name="lblAccountsTitle">
                        <property name="text">
                            <string>–°—á–µ—Ç–∞</string>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignCenter</set>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QTableWidget" name="tblAccounts"/>
                </item>
                <item>
                    <widget class="QLabel" name="lblTransactionsTitle">
                        <property name="text">
                            <string>–¢—Ä–∞–Ω–∑–∞–∫—Ü–∏–∏</string>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignCenter</set>
                        </property>
                    </widget>
                </item>

                <!-- üîç –ü–æ–∏—Å–∫ –ø–æ —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏—è–º -->
                <item>
                    <layout class="QHBoxLayout" name="layoutSearch">
                        <item>
                            <widget class="QLineEdit" name="searchEdit">
                                <property name="placeholderText">
                                    <string>üîç –ü–æ–∏—Å–∫ –ø–æ —Ç—Ä–∞–Ω–∑–∞–∫—Ü–∏—è–º...</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="btnClearFilter">
                                <property name="text">
                                    <string>–°–±—Ä–æ—Å–∏—Ç—å —Ñ–∏–ª—å—Ç—Ä</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>

                <item>
                    <widget class="QTableWidget" name="tblTransactions"/>
                </item>

                <item>
                    <widget class="QTableWidget" name="tblTransactions"/>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>